src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((eachLine) => eachLine.trim())
+    .filter((line) => line.includes(':'))
+    .reduce((accumulator, line) => {
+      const [styleName, value] = line.split(':');
```

Comment:
The code correctly splits each line into a style name and value. However, it does not handle cases with multiple colons within the property values. While this is not required for the task, it's worth noting that in a more robust implementation, you might want to split only on the first colon to avoid issues with property values that contain colons, such as URLs in `background-image` properties.